js/main.js

```diff
@@ -309,7 +315,144 @@ function createTrainerEntry(index){
+  if (stats.name === "nidoran-m" || stats.name === "nidoran-f"){
+    $nameAndNumber.textContent = 'Nidoran' + " (#" + stats.id + ")"
+  } else if (stats.name.includes("-")) {
+    var noSpaces = stats.name.split('-')
+    var answer = [];
+    for (var i = 0; i < noSpaces.length; i++){
+      answer.push(noSpaces[i].charAt(0).toUpperCase() + noSpaces[i].slice(1));
+    }
+    var upperCasedPokemon = answer.join('-')
+    $nameAndNumber.textContent = upperCasedPokemon + " (#" + stats.id + ")"
+  } else {
+    $nameAndNumber.textContent = stats.name.charAt(0).toUpperCase() + stats.name.slice(1) + " (#" + stats.id + ")"
+  }
+  $nameAndNumber.className = "pokedexh1"
+  $pokedexImage.appendChild($nameAndNumber)
+  var $ul = document.createElement('ul');
+  for (var i = 0; i < stats.stats.length; i++){
+    var $li = document.createElement('li');
+    $li.textContent = stats.stats[i].base_stat + " " + stats.stats[i].stat.name;
+    $ul.appendChild($li);
+  }
+  for (var i = 0; i < stats.types.length; i++){
+    var $h1 = document.createElement('h1');
+    var $divH1 = document.createElement('div');
+    if (stats.types[i].type.name === "normal"){
+      $divH1.className = "typeColor typeNormal"
+    } else if (stats.types[i].type.name === "fighting"){
+      $divH1.className = "typeColor typeFight"
+    } else if (stats.types[i].type.name === "flying") {
+      $divH1.className = "typeColor typeFlying"
+    } else if (stats.types[i].type.name === "poison") {
+      $divH1.className = "typeColor typePoison"
+    } else if (stats.types[i].type.name === "ground") {
+      $divH1.className = "typeColor typeGround"
+    } else if (stats.types[i].type.name === "rock") {
+      $divH1.className = "typeColor typeRock"
+    } else if (stats.types[i].type.name === "bug") {
+      $divH1.className = "typeColor typeBug"
+    } else if (stats.types[i].type.name === "ghost") {
+      $divH1.className = "typeColor typeGhost"
+    } else if (stats.types[i].type.name === "steel") {
+      $divH1.className = "typeColor typeSteel"
+    } else if (stats.types[i].type.name === "fire") {
+      $divH1.className = "typeColor typeFire"
+    } else if (stats.types[i].type.name === "water") {
+      $divH1.className = "typeColor typeWater"
+    } else if (stats.types[i].type.name === "grass") {
+      $divH1.className = "typeColor typeGrass"
+    } else if (stats.types[i].type.name === "electric") {
+      $divH1.className = "typeColor typeElectric"
+    } else if (stats.types[i].type.name === "psychic") {
+      $divH1.className = "typeColor typePsychic"
+    } else if (stats.types[i].type.name === "ice") {
+      $divH1.className = "typeColor typeIce"
+    } else if (stats.types[i].type.name === "dragon") {
+      $divH1.className = "typeColor typeDragon"
+    } else if (stats.types[i].type.name === "dark") {
+      $divH1.className = "typeColor typeDark"
+    } else if (stats.types[i].type.name === "fairy") {
+      $divH1.className = "typeColor typeFairy"
+    }
```

Comment:
If you just change the class name to be the same as the value from the API (aka `typeGround` becomes just `ground`), you can just do `$divH1.className = "typeColor " + stats.types[i].type.name`

No conditional necessary 😎